care/emr/api/viewsets/scheduling/token.py

```
@@ -122,13 +128,19 @@ def authorize_create(self, instance):
             resource,
             self.request.user,
         ):
-            raise PermissionDenied("You do not have permission to create token queue")
+            raise PermissionDenied("You do not have permission to create token")
 
     def authorize_update(self, request_obj, model_instance):
```

Ruff (0.14.4)

Unused method argument: `request_obj`

(ARG002)